main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       knob
+//  action 1:   rotates clockwise
+knob.rotateCw();
+//  action 2:   rotates counterclockwise
+knob.rotateAcw();
+//  action 3:  when pushed, triggers a switch
+knob.switch();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       monitor
+//  action 1:   turns on/off
+monitor.switch();
+//  action 2:   adjusts brightness
+monitor.brightness();
+//  action 3:   adjusts refresh rate
+monitor.refreshRate();

```

Comment:
`switch` is an object.  your action was `turns on/off`.  the verb used is `turns`
`brightness` is a noun or property.  it is not an action.  the verb used is `adjusts`
`refreshRate` is a noun or property.  it is not an action.  the verb used is `adjusts`
- [ ] revise